src/ballet/base64/Local.mk

```diff
@@ -0,0 +1,3 @@
+$(call add-hdrs,fd_base64.h)
```

Comment:
This is test only code right? Prefer to have this in a test lib than shipping as part of ballet.

Reply:
Good catch I'll follow up with ya offline on that.